binstar_client/utils/conda.py

```diff
@@ -19,7 +19,11 @@ def _import_conda_root():
 
 def _conda_root_from_conda_info():
     try:
-        output = subprocess.check_output([CONDA_EXE, 'info', '--json']).decode("utf-8")
+        command = CONDA_EXE
```

Comment:
Perhaps we can create a _get_conda_exe function to encapsulate this functionality, and try to fail gracefully on _conda_root_from_conda_info if we can't find the executable.